test/Artists.js

```diff
@@ -0,0 +1,42 @@
+const { expectRevert } = require('@openzeppelin/test-helpers');
+
+const Contract = artifacts.require('Artists');
```

Comment:
Style normally has the contract name as Artists over Contract, but not a big one